src/nvim/testdir/setup.vim

```diff
@@ -21,7 +21,10 @@ set undodir^=.
 set wildoptions=
 set startofline
 set sessionoptions+=options
+
+" Unmap default mappings
```

Comment:
i intentionally kept it with the group headed by the comment `" Align Nvim defaults to Vim.`

Reply:
Ah okay. I was concerned about future devs (or even my future self) wondering why those two lines were there and thought an explanatory comment could help.